packages/kotti-ui/source/kotti-line/KtLine.vue

```diff
@@ -1,23 +1,46 @@
 <template>
 	<div class="kt-line">
-		<div class="kt-line__line" />
-		<div v-if="text" class="kt-line__text" v-text="text" />
-		<div v-if="text" class="kt-line__line" />
+		<div v-if="position !== Position.LEFT" class="kt-line__line" />
+		<div v-if="text" :class="textClasses" @click="handleClick" v-text="text" />
+		<div v-if="position !== Position.RIGHT" class="kt-line__line" />
 	</div>
 </template>
 
 <script lang="ts">
-import { defineComponent } from '@vue/composition-api'
+import { computed, defineComponent } from '@vue/composition-api'
 
 import { KottiLine } from './types'
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const validatePosition = (value: any): value is KottiLine.Position =>
```

Comment:
can work with `unknown`

Reply:
Actually, in most cases that yields a lot of annoying type errors in those validation functions 😔

Agree with it on an idealistic level though, would be nice if it could be used as it's the best fit, just not practically feasible

Reply:
typescript will actually say that `.includes()` is not allowed because value is not of type `Position` :(